admin_guide/pruning_resources.adoc

```diff
@@ -209,8 +222,18 @@ is younger than `<duration>` relative to the current time. (default *60m*)
 |Prune each image that exceeds the smallest xref:limits.adoc#image-limits[limit]
 defined in the same project. This flag cannot be combined with `--keep-tag-revisions`
 nor `--keep-younger-than`.
+
+.^|`--registry-url`
+|The address to use when contacting the registry. The command will attempt to
+use a cluster-internal URL determined from managed images and image streams. In
+case it fails (the registry cannot be resolved or reached), an alternative route
+that works needs to be provided using this flag. If provided, a secure
+connection will be initiated. To allow a fall-back to insecure transport, use
```

Comment:
Don't encourage using `--force-insecure` anywhere, so drop the last sentence. It's already explained and that's it, it should not be used anywhere, though.